src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs

```diff
@@ -135,29 +136,30 @@ public async Task<ViewResult> SelectApprenticeships(string accountId, string acc
 
         [HttpPost]
         [Route("{accountLegalEntityId}/select-new-apprentices")]
-        public async Task<IActionResult> SelectApprenticeships(string accountId, string accountLegalEntityId, [FromBody] SelectApprenticeshipsViewModel viewModel)
+        public async Task<IActionResult> SelectApprenticeships(SelectApprenticeshipsRequest form)
         {
-            if (viewModel.HasSelectedApprenticeships)
+            if (form.HasSelectedApprenticeships)
             {
-                return RedirectToAction("Declaration", new { accountId });
+                return RedirectToAction("Declaration", new { form.AccountId });
             }
 
+            var viewModel = await GetInitialSelectApprenticeshipsViewModel(form.AccountId, form.AccountLegalEntityId);
             ModelState.AddModelError(viewModel.FirstCheckboxId, SelectApprenticeshipsViewModel.SelectApprenticeshipsMessage);
```

Comment:
This will error if there are no apprenticeships.  Rather than use FirstCheckBoxId just use a constant for the key.

Reply:
constant? the requirement from Greg is to navigate & highlight the first checkbox. its ID is not a constant.